modules/nextflow/src/main/groovy/nextflow/processor/TaskConfig.groovy

```diff
@@ -420,6 +420,10 @@ class TaskConfig extends LazyMap implements Cloneable {
         return CmdLineOptionMap.emptyOption()
     }
 
+    Map<String, String> getStickers() {
+        return (get('sticker') ?: Collections.emptyMap()) as Map<String, String>
+    }
+
```

Comment:
Nice idea, but the `sticker` name would introduce a new concept, thats' something we are trying to prevent. Let's rename this to `resourceLabels` / `getResourceLabels`.

Reply:
Ok, but this is "transparent" for the DSL, it uses `label` as usual and internally we maintain in a new concept

Reply:
that's true, but it also a good thing keep the internal domain names consistent with the dsl